Update draft-cwbgp-ivy-energy-saving-management.md (#17)
boucadair/draft-cwbgp-energy-saving-management: draft-cwbgp-ivy-energy-saving-management.md

   environmental impact, it is important to ensure energy efficiency in
   the operation of network infrastructures.  Operators are thus seeking
   for more information to reflect the power consumption of a network

>    and the contribution of involved nodes. As described in {{!RFC6988}},

Reviewer: ```suggestion
   and the contribution of involved nodes. As described in {{Section 3.4 of ?RFC6988}},
```

>    monitoring energy, power can be required for purposes such as:

Reviewer: ```suggestion
   monitoring energy, power can be required for purposes such as:
   
```

>     o build close loops that save energy

Reviewer: ```suggestion
    o designing control loops for energy saving
```

    o investigating energy-saving potential
    o evaluating the effectiveness of energy-saving policies and
      measures
    o accounting for the total power received and provided by an entity,
      a network, or a service
    o predicting an entity's reliability based on power usage

>     o choosing the time of the next maintenance cycle for an entity

Reviewer: ```suggestion
    o planning for the next maintenance cycle for an entity
```
